code-of-conduct.html

```diff
@@ -0,0 +1,93 @@
+          <p>No weapons of any kind,  or drugs are allowed at Write/Speak/Code.</p>
+        </div>
+        <div class="long-version">
+          <h3>The Less Quick Version</h3>
+          <p>Write/Speak/Code is dedicated to providing a harassment-free experience for everyone. We do not tolerate harassment of participants in any form. This code of conduct applies to all Write/Speak/Code spaces both online and off. Anyone who violates this code of conduct may be sanctioned or expelled from these spaces at the discretion of the Write/Speak/Code.</p>
+          <p>Some Write/Speak/Code spaces may have additional rules in place, which will be made clearly available to participants. Participants are responsible for knowing and abiding by these rules.</p>
+
+          <h4>Harassment includes:</h4>
+          <ul>
+            <li>Offensive comments related to gender, gender identity and expression, sexual orientation, disability, mental illness, neuro(a)typicality, physical appearance, body size, age, race, or religion.</li>
+            <li>Unwelcome comments regarding a person’s lifestyle choices and practices, including those related to food, health, parenting, drugs, and employment.</li>
+            <li>Deliberate misgendering or use of ‘dead’ or rejected names.</li>
+            <li>Gratuitous or off-topic sexual images or behaviour in spaces where they’re not appropriate.</li>
+            <li>Physical contact and simulated physical contact (eg, textual descriptions like “*hug*” or “*backrub*”) without consent or after a request to stop.</li>
+            <li>Threats of violence.</li>
+            <li>Incitement of violence towards any individual, including encouraging a person to commit suicide or to engage in self-harm.</li>
+            <li>Deliberate intimidation.</li>
+            <li>Stalking or following.</li>
+            <li>Harassing photography or recording, including logging online activity for harassment purposes.</li>
+            <li>Sustained disruption of discussion.</li>
+            <li>Unwelcome sexual attention.</li>
+            <li>Pattern of inappropriate social contact, such as requesting/assuming inappropriate levels of intimacy with others.</li>
+            <li>Continued one-on-one communication after requests to cease.</li>
+            <li>Deliberate “outing” of any aspect of a person’s identity without their consent except as necessary to protect vulnerable people from intentional abuse.</li>
+            <li>Publication of non-harassing private communication.</li>
+          </ul>
+
+          <h4>Write/Speak/Code prioritizes marginalized people’s safety over privileged people’s comfort and therefore we will not act on complaints regarding:</h4>
+          <ul>
```

Comment:
the  `<h4>` section here is a bit hard to read, maybe remove the capitalization or change the line height or... whatever looks good